roles/conjur_host_identity/tests/test_cases/misconfig-conjur-identity/misconfig-role.yml

```diff
@@ -0,0 +1,21 @@
+---
```

Comment:
Some notes re. this playbook:
- We probably don't want to `ignore_errors`, as that's what we're trying to validate.
- I think we should repurpose this test to confirm a more specific failure case specific to the `conjur_host_identity` role. For example, the playbook should fail [if a required variable is missing](https://github.com/cyberark/ansible-conjur-collection/blob/main/roles/conjur_host_identity/tasks/identity_check.yml#L11).
  - If we want to include another test case to confirm how the role responds to a `401 Unauthorized` response from Conjur, a more applicable approach might be to provide a malformed HF token, instead of an invalid API endpoint.
- We might want to consider using a `block/rescue` pattern, which will make it more explicit that the playbook is _expected_ to fail.

```
---
- name: Configuring Conjur identity on remote hosts fails when missing required variable
  hosts: testapp
  tasks:
    - name: Attempt to configure Conjur identity
      block:
        - import_role:
            name: "cyberark.conjur.conjur-host-identity"
          vars:
            conjur_account: cucumber
            # conjur_appliance_url: "https://conjur-proxy-nginx"
            conjur_host_factory_token: "{{lookup('env', 'HFTOKEN')}}"
            conjur_host_name: "conjur_{{ ansible_hostname }}"
            conjur_ssl_certificate: "{{lookup('file', '../../conjur.pem')}}"
            conjur_validate_certs: yes
      rescue:
        - name: Confirm Role setup fails
          assert:
            that: ansible_failed_result.failed == true
        - name: Confirm error message
          assert:
            that: ansible_failed_result.msg == "'conjur_appliance_url' is undefined"
```